libpod/runtime_img.go

```diff
@@ -96,30 +173,35 @@ func (r *Runtime) PullImage(imgName string, allTags bool, signaturePolicyPath st
 
 	// PullImage copies the image from the source to the destination
 	var (
-		images []string
+		images       []string
+		fromRegistry bool
```

Comment:
(`fromRegistry == (srcRef == nil)`, I’m not sure that having a redundant boolean is more readable. But this will have to be different anyway…)

Reply:
removed `fromRegistry`. Wasn't needed anymore